wpilibNewCommands/src/main/native/cpp/frc2/command/CommandPtr.cpp

```diff
@@ -165,6 +165,16 @@ CommandPtr CommandPtr::OnlyIf(std::function<bool()> condition) && {
   return std::move(*this).Unless(std::not_fn(std::move(condition)));
 }
 
+CommandPtr CommandPtr::After(std::function<bool()> condition) && {
+  AssertValid();
+  return std::move(*this).BeforeStarting(std::make_unique<WaitUntilCommand>(std::move(condition)));
```

Comment:
```suggestion
  return std::move(*this).BeforeStarting(cmd::WaitUntil(condition));
```
(`BeforeStarting()` takes a `CommandPtr`, which does not have an implicit conversion from `std::unique_ptr`)
If you really want to write it in terms of constructing a `WaitUntilCommand`, you should be able to do this:
```suggestion
  return std::move(*this).BeforeStarting(WaitUntilCommand(condition).ToPtr());
```